GeneratorInterface/Pythia8Interface/plugins/Pythia8Hadronizer.cc

```diff
@@ -309,6 +313,17 @@ Pythia8Hadronizer::Pythia8Hadronizer(const edm::ParameterSet &params)
                                                    0));
   }
 
+  if (params.exists("UserCustomization")) {
+    fCustomHooksVector.reset(new UserHooksVector);
```

Comment:
i don't think this line is necessary

Reply:
Without resetting it here it goes to a segmentation fault because fCustomHooksVector is not initalized

Reply:
Ah, it's for initialization. In that case, I would rewrite it this way (avoiding `new` makes it more obvious that the memory is being managed):
```cpp
fCustomHooksVector = std::make_shared<UserHooksVector>();
```

Reply:
ok!